.github/workflows/ci.yaml

```diff
@@ -20,22 +21,17 @@ jobs:
         run: npm run check
       - name: Build
         run: npm run build
-      - name: Upload pages artifact
-        uses: actions/upload-pages-artifact@v3
-        with:
-          path: dist
 
-  deploy:
-    needs: build
+ 
+  webhook-live:
     runs-on: ubuntu-latest
-    permissions:
-      pages: write      # to deploy to Pages
-      id-token: write   # to verify the deployment originates from an appropriate source
-    environment:
-      name: github-pages
-      url: ${{ steps.deployment.outputs.page_url }}
-    if: github.ref == 'refs/heads/main'
+    needs: build
+    if: ${{ success() }}
     steps:
-      - name: Deploy to GitHub Pages
-        id: deployment
-        uses: actions/deploy-pages@v4
\ No newline at end of file
+    - name: Workflow Webhook Action Live
+      uses: distributhor/workflow-webhook@v2.0.3
+      env:
+        webhook_type: 'json-extended'
+        webhook_url: ${{ secrets.WEBHOOK_URL_LIVE }}/root/dev-tools/webhooks/dilewe_pipeline_live.sh
```

Comment:
Why is the path to a script in here? Normally one just gives a name to an endpoint which then runs a script specified in a config file, since running arbitrary code/scripts based on HTTP Requests would be a massive security issue.

Reply:
As per your comment in the other PR:

> 
[...] still pondering whether the secret should contain the entire URL or just the domain, with the path specified in the yaml file. Kinda leaning towards the latter one since it'd make it simpler to create new endpoints and the path can be easily guessed anyways.

Otherwise would be great if we can take a look together at this :)

Reply:
See other pr, endpoint is created in webhooks .json